tools/appslib/logging_sender.py

```diff
@@ -1,16 +1,37 @@
 #!/usr/bin/env python3
 
 import subprocess
-from shutil import which
 import logging
 import logging.handlers
 
 
-def send_to_matrix(message: str) -> None:
-    if which("sendxmpppy") is None:
-        logging.warning("Could not send error via xmpp.")
-        return
-    subprocess.call(["sendxmpppy", message], stdout=subprocess.DEVNULL)
+def notify(message, channel):
+    print(f"{channel} -> {message}")
+
+    chan_list = ["dev", "apps", "doc"]
+
+    if not any(channel in x for x in chan_list):
+        logging.error(
+            f"Provided chan '{channel}' is not part of the available options ('dev', 'apps', 'doc')."
+        )
+
+    for char in ["'", "`", "!", ";", "$"]:
+        message = message.replace(char, "")
```

Comment:
Hmmmm why is that ? Couldn't we escape them ? Or store the message in a file passed to matrix-commander to prevent this kind of issue ?

Reply:
don't know, took from there https://github.com/YunoHost/webhooks/blob/master/server.py?rgh-link-date=2024-03-15T03%3A43%3A50Z#L34

Reply:
weird, but ok…